gemrb/GUIScripts/pst/GUIOPT.py

```diff
@@ -159,13 +132,11 @@ def GammaFeedback (feedback_ref):
 
 def OpenAudioOptionsWindow ():
 	"""Open audio options window"""
-	global SubOptionsWindow, AudioHelpText
-
-	GemRB.HideGUI ()
+	global AudioHelpText
```

Comment:
Codacy found an issue: [Global variable 'AudioHelpText' undefined at the module level](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)